packages/core/examples/colorbar/index.ts

```diff
@@ -0,0 +1,728 @@
+import vtkColormaps from '@kitware/vtk.js/Rendering/Core/ColorTransferFunction/ColorMaps';
+import {
+  RenderingEngine,
+  Types,
+  Enums,
+  volumeLoader,
+  getRenderingEngine,
+  ui,
+} from '@cornerstonejs/core';
+import * as cornerstoneTools from '@cornerstonejs/tools';
+import {
+  initDemo,
+  createImageIdsAndCacheMetaData,
+  setTitleAndDescription,
+  addDropdownToToolbar,
+  setCtTransferFunctionForVolumeActor,
+  setPetColorMapTransferFunctionForVolumeActor,
+} from '../../../../utils/demo/helpers';
+
+const { ViewportColorBar } = ui.widgets.colorbar;
```

Comment:
I feel like the colorbar belongs into tools not core. 
What to call it? hmmm maybe we just put it under utilities.voi and no mention of ui or widget